bundles/org.eclipse.passage.lbc.jetty/src/org/eclipse/passage/lbc/internal/jetty/JettyServerLauncher.java

```diff
@@ -66,27 +68,33 @@ public LicensingResult launch(Map<String, Object> arguments) {
 			server.setHandler(new JettyRequestHandler(requestDispatchers.values()));
 			server.start();
 			logger.info(server.getState());
-			return LicensingResults.createOK(JettyMessages.JettyServerLauncher_ok_start, source);
+			return LicensingResults.createOK(JettyMessages.JettyServerLauncher_server_start_success, source);
 		} catch (Exception e) {
-			logger.warn(e.getMessage());
-			return LicensingResults.createError(JettyMessages.JettyServerLauncher_e_start, source, e);
+			String msg = e.getMessage();
```

Comment:
You should formulate your own message here: the one extracted from the exception will not be helpful